anyway/widgets/all_locations_widgets/most_severe_accidents_table_widget.py

```diff
@@ -44,19 +44,9 @@ def get_most_severe_accidents_table_title(
     location_info: dict, resolution: BE_CONST.ResolutionCategories
 ):
     if resolution == BE_CONST.ResolutionCategories.SUBURBAN_ROAD:
-        return (
-            _("Most severe accidents in segment")
-            + " "
-            + segment_dictionary[location_info["road_segment_name"]]
-        )
+        return _("Most severe accidents in segment"), segment_dictionary[location_info["road_segment_name"]]
```

Comment:
Localization should only be done in the `localize_items()` method. The `generate_items()` method should generate the texts in English.